packages/@react-spectrum/dropzone/src/DropZone.tsx

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2023 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {AriaLabelingProps, DOMProps, DOMRef, StyleProps} from '@react-types/shared';
+import {classNames, SlotProvider, useDOMRef, useStyleProps} from '@react-spectrum/utils';
+import {DropZoneProps, DropZone as RACDropZone} from 'react-aria-components';
+import {mergeProps, useId} from '@react-aria/utils';
+import React, {ReactNode} from 'react';
+import styles from '@adobe/spectrum-css-temp/components/dropzone/vars.css';
+
+export interface SpectrumDropZoneProps extends DropZoneProps, DOMProps, StyleProps, AriaLabelingProps {
```

Comment:
Might be some props we don't need here (DropZone's slotProps and the extra AriaLabelingProps) but we can double check when we see the PropsTable in the docs